zellij-server/src/os_input_output.rs

```diff
@@ -29,6 +33,7 @@ use zellij_utils::{
 
 use async_std::io::ReadExt;
 pub use async_trait::async_trait;
+use byteorder::{BigEndian, ByteOrder};
```

Comment:
What are we using`ByteOrder` for?

Reply:
The Unix pipe API we use to send the terminal process id back to our parent process needs a `&[u8]`. So we have to encode the u32 process id returned by the std::process::Child into a `&[u8; 4]` array to send it. The byteorder crate does that for us, `ByteOrder` is just the trait that implements the method to do so.